src/Utils/obiUtils.tsx

```diff
@@ -653,6 +653,9 @@ export async function importActionOutput(
             if (!entityId) {
                 throw new Error("Invalid Entity Definition")
             }
+            // Record the created entity in the input entity list.
+            newEntity.entityId = entityId
+            entities.push(newEntity)
```

Comment:
Hmm...  I think it would be better to create a local list rather than push these on to the passed list.  That could have unintended side effects